traffic-id.yaml

```diff
@@ -55,6 +55,21 @@ tls-sni-patterns:
     patterns:
       - googlevideo.com
 
+  - id: google-tracking
+    labels:
+      - tracking
+      - stats
```

Comment:
We had discussed trying to stick close to Qosmos, or other tools that had already done some labelling. In which case, "stats" would be "analytics". They don't have one for tracking, so I think tracking is good.